ci/spec.yml

```diff
@@ -134,19 +134,18 @@ jobs:
       - *copy_secret
 
       - name: Regenerate workflow YAML file and check for changes
-        run: |
-          set -e
-          pip install pyyaml
-          ./scripts/build_ensure_no_changes.sh ./ci/regen-workflow.py
+        run: ./scripts/build_ensure_no_changes.sh ./ci/regen-workflow.py
```

Comment:
This is already a part of #1143.

Reply:
Yup, I figured I'd just rebase whichever one goes in second as otherwise I'd have to muck around w/ downgrading my local setuptools to work around this.